src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/SdkTemplateTest.cs

```diff
@@ -29,6 +29,14 @@ internal void Execute(DotNetSdkHelper dotNetHelper, string testRoot, string[]? f
         string projectName = $"{ScenarioName}_{Template}_{Language}";
         string customNewArgs = Template.IsAspNetCore() && NoHttps ? "--no-https" : string.Empty;
         string projectDirectory = Path.Combine(testRoot, projectName);
+        string portableRid = $"linux-{TargetArchitecture}";
+
+        // We don't need to separately execute with the portable rid when it is the same as the TargetRid.
+        // When they are different (that is: the build is non-portable), the portable assets needed for the tests were produced by a different build.
+        // These portable assets are not available for community supported architectures.
+        bool executeWithPortableRid = TargetRid != portableRid
+                                      && TargetArchitecture != "s390x"
+                                      && TargetArchitecture != "ppc64le";
```

Comment:
Should we proactively add all non-Microsoft architectures from https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.architecture?view=net-8.0#fields to here?

Reply:
I've included the ones that are causing test failures for us. Can you list what additional ones I should add?

Reply:
Looking at https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.architecture?view=net-9.0#fields : 

- `loongarch64`
- `riscv64`